### PR TITLE
Added .destroy() function with tests for File-Bin

### PR DESCRIPTION
**Pull request by acareaga**
Implemented the destroy (aka delete.. ?) for file-bin. 

Please let me know if you see any errors or areas that should be improved

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -57,6 +57,15 @@ FileBin.prototype.write = function (fileName, data) {
   });
 };
 
+FileBin.prototype.destroy = function (fileName) {
+  return new RSVP.Promise((resolve, reject) => {
+    fs.unlink(path.join(this.base, fileName), (error) => {
+      if (error) { return reject(error); }
+      resolve(formatFile(fileName));
```
Comment (stevekinney) [inline index.js]: It's kind of weird to return our now dead file. What about `resolve(true)` instead?

**Comment by stevekinney** [comment]
Can you add some documentation to the README and squash the commits?

**Comment by stevekinney** [comment]
We also have some conflicts now.

**Comment by acareaga** [comment]
Sounds good - I'll fix those conflicts, clean up the commits, and add some documentation tonight. 

Resolve true makes more sense. I'm still a little confused with the promise structure.

**Comment by acareaga** [comment]
I have to be missing something... tried for over an hour to squash previous commits. Git says I picked and squashed the correct files and rebased to the most recent commit locally but the PR still shows 10 commits.

**Comment by stevekinney** [comment]
Force push?

**Pull request closed without merging**